pyccel/commands/lambdify.py

```diff
@@ -14,7 +14,8 @@
     from sympy.printing.pycode import NumPyPrinter
 
 def lambdify(expr : sp.Expr, args : 'dict[sp.Symbol, str]', *, result_type : str = None,
-             templates : 'dict[str,list[str]]' = None, **kwargs):
+             templates : 'dict[str,list[str]]' = None, collect_result_in_arg = False,
+             **kwargs):
```

Comment:
More intuitive argument name:
```suggestion
             templates : 'dict[str, list[str]]' = None, use_out = False,
             **kwargs):
```

Reply:
Done [c36d1c6](https://github.com/pyccel/pyccel/pull/1867/commits/c36d1c6cc5a0f652db726df919008d3bfbbb4f43)